### ISSUE TITLE
[cli] `hasura console` loads remote assets depite config

### ISSUE THREAD

**Issue by haf** [hasura/graphql-engine#8482]
### Version Information

Server Version: 2.6.1
CLI Version (for CLI related issue): 2.6.1

### Environment

docker

### What is the current behaviour?

I expect that 

    HASURA_GRAPHQL_CONSOLE_ASSETS_DIR: /srv/console-assets

would only serve assets from the docker container.

### What is the expected behaviour?

It serves from the CDN (so I can't work offline)

<img width="1800" alt="image" src="https://user-images.githubusercontent.com/193115/167894208-b92798e6-8c45-4290-bf1b-8675651640d0.png">

### Keywords

console remote cdn serve request remote

### How to reproduce the issue?

1. start the console with the above config on the server

**Comment by jflambert** [comment]
Interesting. I can't replicate with 2.6.1

**Comment by haf** [comment]
I even did `docker compose exec api printenv` to validate that the env var was set, and I did a hard refresh in the browser also to verify. I'm on that version, too.

**Comment by jflambert** [comment]
Here's my test docker-compose.yaml file

```
version: '3.6'

volumes:
  db_data:

services:
  postgres:
    image: timescale/timescaledb-ha:pg13.6-ts2.6-latest
    ports:
      - "5432:5432"
    restart: always
    volumes:
      - db_data:/var/lib/postgresql/data
    environment:
      POSTGRES_DB: testdb
      POSTGRES_PASSWORD: postgrespassword
      TIMESCALEDB_TELEMETRY: "off"

  graphql-engine:
    image: hasura/graphql-engine:v2.6.1
    ports:
      - "8080:8080"
    depends_on:
      - "postgres"
    restart: always
    environment:
      HASURA_GRAPHQL_DATABASE_URL: postgres://postgres:postgrespassword@postgres:5432/testdb?application_name=hasura
      HASURA_GRAPHQL_ENABLE_CONSOLE: "true"
      HASURA_GRAPHQL_ENABLE_TELEMETRY: "false"
      HASURA_GRAPHQL_LOG_LEVEL: "warn"
      HASURA_GRAPHQL_EXPERIMENTAL_FEATURES: "inherited_roles"
      HASURA_GRAPHQL_CONSOLE_ASSETS_DIR: "/srv/console-assets"
```

and here's my "Sources" tab in Chrome's inspector.

![image](https://user-images.githubusercontent.com/11539524/168067481-20c38aa5-4579-4376-b4d2-9a367f378cbf.png)

**Comment by haf** [comment]
Let me check locally and get back to you. Maybe it’s particular to Firefox

**Comment by haf** [comment]
Same for me:

```
  api:
    image: hasura/graphql-engine:v2.6.1
    restart: unless-stopped
    ports:
    - 8080:8080
    depends_on:
    - db
    networks:
      app:
        aliases:
        - analytics-db
        - analytics-db.app.svc
        - analytics-db-m2m
        - analytics-db-m2m.app.svc
    environment:
      HASURA_GRAPHQL_DATABASE_URL: postgres://postgres:abc123456789@db.app.svc:5432/analytics
      HASURA_GRAPHQL_ENABLE_CONSOLE: "true"
      HASURA_GRAPHQL_CONSOLE_ASSETS_DIR: "/srv/console-assets"
      HASURA_GRAPHQL_DEV_MODE: "true"
      HASURA_GRAPHQL_ENABLED_LOG_TYPES: startup, http-log, webhook-log, websocket-log, query-log
      HASURA_GRAPHQL_ADMIN_SECRET: 638BD5723C2E4A5AB2AE28CF5FCEC283 # see ../../analytics-db/{Makefile,k8s/dev}
      HASURA_GRAPHQL_ENABLE_TELEMETRY: "false"
      # https://manage.auth0.com/dashboard/eu/causiq/applications/MfTEaXZO77MWGCikVngpLXL4sCi2CNBM/settings
      HASURA_GRAPHQL_JWT_SECRET: '{ "type": "RS512", "jwk_url": "$AUTH0_ISSUER_BASE_URL/.well-known/jwks.json" }'

```

```
❯ hasura version
INFO hasura cli                                    version=v2.6.1
INFO hasura graphql engine                         endpoint="http://localhost:8080" version=v2.6.1
❯ hasura console
INFO A new version (v2.6.2) is available for CLI, update? (y/N)
n
INFO skipping update, run 'hasura update-cli' to update manually
INFO console running at: http://localhost:9695/
```

<img width="1201" alt="image" src="https://user-images.githubusercontent.com/193115/168552663-5094d364-2908-4500-a8ac-661c485b3d8c.png">

Also verified with printenv:

```
❯ docker compose exec api printenv | grep DIR
HASURA_GRAPHQL_CONSOLE_ASSETS_DIR=/srv/console-assets
```

**Comment by haf** [comment]
It also disregards this flag `HASURA_GRAPHQL_ENABLE_TELEMETRY: "false"` and sends data from my laptop to your servers anyway.

**Comment by haf** [comment]
Same for v2.6.2, just tried.

**Comment by scriptonist** [comment]
@haf

> It also disregards this flag HASURA_GRAPHQL_ENABLE_TELEMETRY: "false" and sends data from my laptop to your servers anyway.

^ this shouldn't be the case. Can you help me understand how you reached that conclusion?

**Comment by haf** [comment]
Hi, I'm sorry, but for me this issue has timed out and I'm not working on/with Hasura right now.

**Comment by rikinsk** [comment]
Reopening to keep track of the reported behaviour

**Comment by scriptonist** [comment]
This behaviour can be reproduced if Hasura is started with an `admin-secret`. And, subsequent `hasura console` with offline assets configuration will not work.

**Comment by jflambert** [comment]
ah nice find.

**Comment by haf** [comment]
Fixed in v2.17.1-beta.1
